docs/.vuepress/helpers.js

```diff
@@ -64,7 +64,7 @@ function getVersions(buildMode) {
  * @returns {string[]}
  */
 function getFrameworks() {
-  return ['javascript', 'react', 'angular', 'vue2', 'vue'];
+  return ['javascript', 'react', 'angular', 'vue2', 'vue3'];
```

Comment:
@wszymanski I'm wondering if we should do the `vue2` + `vue3` naming or rather stick to our current scheme of `vue` + `vue3` (as in `@handsontable/vue` + `@handsontable/vue3`) WDYT?

Reply:
I did it as in specification, but maybe it can be changed, @warpech WDYT?

Reply:
I didn't notice that's how it's defined in the specification - in that case, let's leave it as is and we can change it on the epic branch if needed.